Agents/FHAgent/sensor/README.md

```diff
@@ -0,0 +1,69 @@
+# Fumehood Occupancy Detection Sensor
+
+This folder contains information regarding the fumehood occupancy sensor.
+
+## Working Details
+
+### Working Principles
+The system uses an ultrasonic proximity sensor to detect the distance between sensor and the closest object.
+If the distance between the sensor and the closest object is below a certain threshold, then the fumehood is considered occupied.
+To prevent noise on the readings, a tally system is used to average the reading data as follows:
+
+1. Set a counter to 0
+2. if the detected distance is below threshold (something nearby the sensor), add a certain increase factor to the counter
+3. Reduce the counter by a certain decrease factor
+4. Ensure the values are within a certain range (counter within tally max amnd tally min)
+5. if the counter is above a tally limit, then the fumehood is considered occupied
+6. Repeat step 2. to 5.
+
+The tally system setup is done on the fumehood agent and currently the agent is sending distance readings to a Thingsboard server where the agent will retrieve the data.
```

Comment:
The fumehood agent is retrieving data from the Thingsboard server right? Not sending distance readings to it.

The proximity sensor is the one sending the distance readings to the Thingsboard server.